### PR TITLE
Install additional libraries

### PR DESCRIPTION
**Pull request by UseinFaradzhev**

### CODE REVIEW

File: infrastructure-provisioning/src/general/scripts/aws/common_install_libs.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/python
```
Comment (VadymKuznetsov) [inline infrastructure-provisioning/src/general/scripts/aws/common_install_libs.py]: Move this script to proper folder

File: infrastructure-provisioning/src/general/scripts/aws/common_list_libs.py [hunk 2]
```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/python
```
Comment (VadymKuznetsov) [inline infrastructure-provisioning/src/general/scripts/aws/common_list_libs.py]: Move this script to proper folder

File: infrastructure-provisioning/src/general/scripts/aws/deeplearning_configure.py [hunk 3]
```diff
@@ -118,6 +118,7 @@
         remove_ec2(notebook_config['tag_name'], notebook_config['instance_name'])
         sys.exit(1)
 
+
```
Comment (VadymKuznetsov) [inline infrastructure-provisioning/src/general/scripts/aws/deeplearning_configure.py]: Why do you need this double enter here?

Comment (Moskovych) [inline infrastructure-provisioning/src/general/scripts/aws/deeplearning_configure.py]: This omission resulting refactoring, which already fixed.

File: infrastructure-provisioning/src/general/scripts/aws/common_install_libs.py (hunk 1 above)
Comment (Moskovych) [inline infrastructure-provisioning/src/general/scripts/aws/common_install_libs.py]: Done.

File: infrastructure-provisioning/src/general/scripts/aws/common_list_libs.py (hunk 2 above)
Comment (Moskovych) [inline infrastructure-provisioning/src/general/scripts/aws/common_list_libs.py]: Done.

**Pull request merged**